### ISSUE TITLE
Support Border Radius for Donut, Pie and Polar Area Charts

### ISSUE THREAD

**Issue by Thimows** [apexcharts/apexcharts.js#4426]
## Summary

A very nice feature would be the ability to set the border radius of the Donut, Pie and Polar Area charts. This would allow us to make charts that look like this:

![Donut Charts Question](https://github.com/apexcharts/apexcharts.js/assets/62159496/251237ef-aec8-4f42-9dfd-4bb1182ccdec)

## API Changes

This setting could live in the plotOptions for the Donut, Pie and Polar Area charts and can be called [borderRadius](https://apexcharts.com/docs/options/plotoptions/bar/#borderRadius) (just like for the bar charts)

**Comment by feroz39** [comment]
It will be appreciated.

**Comment by fornazaric** [comment]
It would really be great, i might try to do a PR with the tools already available in the DrawPath method

**Comment by mustafadilmac** [comment]
This PR is ı guess not accepted yet. I can't figure it out.

**Comment by Nathan7934** [comment]
Man, I was hoping sooo bad that this was already supported 😭

**Comment by KasiViswanath-916** [comment]
it is not working

**Comment by Thimows** [comment]
@junedchhipa Looking at the demand for this feature, it would be amazing if this could be considered 🙂

**Comment by JoePereiraa** [comment]
@KasiViswanath-916 Bro, this hasn't been added yet...

**Comment by fillobotto** [comment]
Up this

**Comment by mulgerehircum** [comment]
Would be really cool to have this...

**Comment by junedchhipa** [comment]
This shipped in **v6.7.0** as `plotOptions.pie.borderRadius`, and it covers pie, donut and polarArea (all three are rendered by the same `Pie` module). Sorry it took a while, and that the thread was auto-closed by the stale bot rather than as done.

```js
plotOptions: {
  pie: {
    borderRadius: 10,  // px, rounds each slice's corners. 0 = sharp (default)
    spacing: 4,        // px gap between adjacent slices, also added in 6.7.0
  }
}
```

The radius is clamped per slice so opposing fillets cannot cross on a thin or narrow slice, and `spacing` insets both edges symmetrically so a slice's mid-angle is preserved, which keeps its data label and hit region where you expect.

Re-tested on **6.8.0** before posting.

Leaving this closed, now with the release recorded. If the clamping behaves oddly on a particular dataset, please open a fresh issue with the series and radius.
